### PR TITLE
fix(image-editor): source not displaying in input

### LINKED ISSUE #8167: Image component source broken
Probably an incorrect property being referenced for the value

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Summary

Screenshot_20230627_140928

## References

closes #8167 

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

File: packages/editor/src/components/properties/ImageNodeEditor.tsx [hunk 1]
```diff
@@ -45,19 +45,30 @@ export const ImageNodeEditor: EditorComponentType = (props) => {
   const imageComponent = useComponent(entity, ImageComponent)
   const errors = getEntityErrors(props.entity, ImageComponent)
 
+  const imageSource =
```
Comment (HexaField) [inline packages/editor/src/components/properties/ImageNodeEditor.tsx:48]: These properties have been removed recently, please make sure you are working off the latest dev version. We just need image.source here

File: packages/engine/src/scene/components/ImageComponent.ts [hunk 2]
```diff
@@ -192,7 +192,7 @@ export function ImageReactor() {
         // TODO: abort load request, pending https://github.com/mrdoob/three.js/pull/23070
       }
     },
-    [image.resource]
+    [entity, source]
```
Comment (HexaField) [inline packages/engine/src/scene/components/ImageComponent.ts:195]: No need to react to entity, just source

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/editor/src/components/properties/ImageNodeEditor.tsx (hunk 1 above)
Comment (aditya-mitra) [reply packages/editor/src/components/properties/ImageNodeEditor.tsx:48]: image

Keeping only 2 options: `imageComponent.resource.value?.source || imageComponent.source.value`

File: packages/engine/src/scene/components/ImageComponent.ts [hunk 3]
```diff
@@ -192,7 +177,7 @@ export function ImageReactor() {
         // TODO: abort load request, pending https://github.com/mrdoob/three.js/pull/23070
       }
     },
-    [entity, source]
+    [image.source.value]
```
Comment (HexaField) [inline packages/engine/src/scene/components/ImageComponent.ts:180]: this should be `image.source` not `image.source.value`

**Review by HexaField — CHANGES_REQUESTED** [review]

**Review by HexaField — APPROVED** [review]

**Pull request merged**